script/test-setup

```diff
@@ -4,8 +4,12 @@
 
 const fs = require('fs')
 const path = require('path')
+const cp = require('child_process')
 const { getUserDataPath } = require('./dist-info')
 
+const output = cp.execSync('git config -l --show-origin', { encoding: 'utf-8' })
+console.log(`${output}\n\n`)
```

Comment:
Dumping the current state of the git config before running the test suite. I've ported this over to #1910 as it helped with investigating config differences between my environment and CI.